content/en/integrations/faq/list-of-api-source-attribute-value.md

```diff
@@ -243,6 +234,7 @@ private: true
 | Openstack                     | OPENSTACK                  |
 | Opsgenie                      | OPSGENIE                   |
 | Opsmatic                      | OPSMATIC                   |
+| Oracle Cloud                  | ORACLECLOUD                |
```

Comment:
this is the only one I a not seeing an integration page for

Reply:
My guess is this is just Oracle. It's the only name with Oracle in the `source.py`.